.rubocop_todo.yml

```diff
@@ -73,7 +73,7 @@ Metrics/AbcSize:
 # Offense count: 416
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
-  Max: 697
+  Max: 698
```

Comment:
tz tz tz 😜

Reply:
Hey, I can't refactor all of OBS at once :laughing: